internal/store/db/db.go

```diff
@@ -27,13 +27,11 @@ var (
 	ErrUnsupportedDatabase = errors.New("unsupported database type")
 )
 
-type Configuration struct {
-	Type         string
-	Host         string
-	Port         int
-	Timeout      int
-	DatabaseName string
-	Username     string
-	Password     string
-	BatchSize    int
+type DatabaseInfo struct {
```

Comment:
This will definitely conflict with https://github.com/edgexfoundry/app-functions-sdk-go/blob/5cd4eaf22e75ca0ba577fb34a09e242a7372a8ae/internal/store/db/db.go#L30

Reply:
Yep, have to merge the PRs in the correct order.